README.md

```diff
@@ -1,3 +1,88 @@
-pat
-===
-[![Build Status](https://travis-ci.org/gorilla/pat.png?branch=master)](https://travis-ci.org/gorilla/pat)
+# pat [![Build Status](https://travis-ci.org/gorilla/pat.png?branch=master)](https://travis-ci.org/gorilla/pat) [![Go Report Card](https://goreportcard.com/badge/github.com/gorilla/pat)](https://goreportcard.com/report/github.com/gorilla/pat)
+
+### How to use?
+
+pat is pretty simple. The documentation lives [here.](http://www.gorillatoolkit.org/pkg/pat)
+
+Here's an example RESTful API:
+
+```go
+package main
+
+import (
+	"log"
+	"net/http"
+
+	"github.com/gorilla/pat"
+)
+
+func homeHandler(wr http.ResponseWriter, req *http.Request) {
```

Comment:
I'd probably make this a little shorter—I don't think there's a need to show as many handlers. 2-3 would suffice to show a parameterized URL, a static URL and a 3 different methods.